sdk/resources/arm-resources/package.json

```
@@ -47,12 +60,37 @@
     "src/**/*.ts",
     "README.md",
     "rollup.config.js",
-    "tsconfig.json"
+    "tsconfig.json",
+    "review/*",
+    "CHANGELOG.md"
   ],
   "scripts": {
-    "build": "tsc && rollup -c rollup.config.js && npm run minify",
-    "minify": "uglifyjs -c -m --comments --source-map \"content='./dist/arm-resources.js.map'\" -o ./dist/arm-resources.min.js ./dist/arm-resources.js",
-    "prepack": "npm install && npm run build"
+    "build": "tsc && rollup -c rollup.config.js && npm run minify && mkdirp ./review &&  npm run extract-api",
```

@sarangan12 
```suggestion
    "build": "tsc && rollup -c 2>&1 && npm run minify &&  npm run extract-api",
```

Also, why we're still doing this minify thing?

---

we probably shouldn't remove `mkdirp ./review` per this issue https://github.com/Azure/autorest.typescript/issues/800

---

@qiaozha The `mkdirp ./review` has already been added to extract-api script. So, It can be removed safely.

---

Code changes complete with PR https://github.com/Azure/autorest.typescript/pull/1104

---

@sarangan12 Got it. Thanks :)